shuffle/cli/src/transactions.rs

```diff
@@ -0,0 +1,290 @@
+                        .last()
+                        .ok_or_else(|| anyhow!("Couldn't get last transaction"))?,
+                )?;
+
+                if old_seq_num != 0_u64 {
+                    txn_array = normalize_txn_array(txn_array.to_vec());
+                }
+
+                old_seq_num = write_out_txns(
+                    txn_array.to_vec(),
+                    last_txn_seq_num,
+                    old_seq_num,
+                    &mut io::stdout(),
+                )?;
+            }
+        }
+    }
+    Ok(())
+}
+
+pub fn normalize_txn_array(mut arr: Vec<Value>) -> Vec<Value> {
+    // Excluding the index of the txn we've already printed out
+    arr.remove(0);
+    arr
+}
+
+pub fn write_out_txns<W: Write>(
+    all_transactions: Vec<Value>,
+    last_txn_seq_num: u64,
+    old_seq_num: u64,
+    mut stdout: W,
+) -> Result<u64> {
+    // Only want to write the first transaction (seq num: 0) or
+    // transactions where the sequence number changes
+    if last_txn_seq_num > old_seq_num {
+        for txn in all_transactions.iter() {
+            write_into(&mut stdout, txn)?;
+            println!();
+            println!();
+        }
+    }
+    Ok(last_txn_seq_num)
+}
+
+pub async fn get_account_transactions_response(
+    client: &Client,
+    path: &Url,
+    start: u64,
+    limit: u64,
+) -> Result<Response> {
+    Ok(client
+        .get(path.as_str())
+        .query(&[("start", start.to_string().as_str())])
+        .query(&[("limit", limit.to_string().as_str())])
+        .send()
+        .await?)
+}
+
+pub async fn get_account_sequence_number(
+    client: &Client,
+    network: &Url,
+    address: AccountAddress,
+) -> Result<u64> {
+    let path =
+        network.join(format!("accounts/{}/resources/", address.to_hex_literal()).as_str())?;
+    let resp = client.get(path.as_str()).send().await?;
+    let json: Vec<Value> = serde_json::from_str(resp.text().await?.as_str())?;
+    parse_json_for_account_seq_num(json)
+}
+
+pub fn parse_txn_for_seq_num(last_txn: &Value) -> Result<u64> {
+    Ok(last_txn["sequence_number"]
+        .to_string()
+        .replace('"', "")
+        .parse::<u64>()?)
+}
+
+pub fn parse_json_for_account_seq_num(json_objects: Vec<Value>) -> Result<u64> {
+    let mut seq_number_string = "";
+    for object in &json_objects {
+        if object["type"]["name"] == "DiemAccount" {
+            seq_number_string = object["value"]["sequence_number"]
+                .as_str()
+                .ok_or_else(|| anyhow!("Invalid sequence number string"))?;
+            break;
+        };
+    }
+    let seq_number: u64 = seq_number_string.parse()?;
+    Ok(seq_number)
+}
+
+fn write_into<W>(writer: &mut W, json: &serde_json::Value) -> Result<()>
+where
+    W: Write,
+{
+    write!(writer, "{}", json)?;
+    Ok(())
+}
+
+#[cfg(test)]
+mod test {
+    use crate::transactions::{
+        normalize_txn_array, parse_json_for_account_seq_num, parse_txn_for_seq_num, write_into,
```

Comment:
Rather than make all these methods public, do
```rust
    use super::*;
```

This is used elsewhere in the codebase: https://github.com/dimroc/diem/blob/8fab7b7f2e131bdcfad475b0509c636e5e1883bf/sdk/transaction-builder/src/error_explain.rs#L33-L35